requirements.txt

```diff
@@ -1,8 +1,15 @@
 numpy>=1.16.6,<1.20
 scipy>=1.2.3,<2
-scikit-image>=0.14.5
+scikit-image==0.16.1
 scikit-learn>=0.20.4
 tensorflow==2.4.1
 jupyter>=1.0.0,<2
 deepcell-tracking>=0.3.1,<0.4.0
 deepcell-toolbox>=0.9.0,<0.10.0
+opencv-python<=3.4.9.31
+spektral==1.0.6
+pydot>=1.4.1
+pydotplus>=2.0.2
+graphviz>=0.14
+einops==0.3.0
+jupyterlab
```

Comment:
I'd prefer to let users install `jupyterlab` if they want it, but not to ship with it by default.

```suggestion
```